tendermint/src/types/time/duration.rs

```diff
@@ -0,0 +1,589 @@
+// The Duration struct is defined in gogoproto v1.3.1 at https://github.com/gogo/protobuf/blob/v1.3.1/protobuf/google/protobuf/duration.proto
+// and https://github.com/protocolbuffers/protobuf-go/blob/v1.34.2/types/known/durationpb/duration.pb.go
+
+const SECONDS_PER_HOUR: i64 = 60 * 60;
```

Comment:
`i64`?

Reply:
Looks strange but it probably leads to fewer casts later on.

Reply:
Okay.